SCAutolib/controller.py

```diff
@@ -184,8 +188,11 @@ def setup_system(self, install_missing: bool, gdm: bool, graphical: bool):
             self.dconf_file.save()
             run('dconf update')
 
-        run(['dnf', 'groupinstall', "Smart Card Support", '-y'])
-        logger.debug("Smart Card Support group in installed.")
+        if os_version != OSVersion.Fedora:
+            run(['dnf', 'groupinstall', "Smart Card Support", '-y'])
+            logger.debug("Smart Card Support group in installed.")
+        else:
+            run(['dnf', 'install', 'esc', 'pcsc-lite-ccid', '-y'])
```

Comment:
I would put here also `opensc` and maybe remove `esc` as I do not think this is used for anything useful in Fedora (I think it does not even work now).